packages/catalog-process/test/db.test.ts

```diff
@@ -742,282 +1053,875 @@ describe("database test", async () => {
         ).rejects.toThrowError(operationForbidden);
       });
 
-      it("should throw notValidDescriptor if the descriptor is in draft state", async () => {
-        const descriptor: Descriptor = {
-          ...mockDescriptor,
-          state: descriptorState.draft,
-        };
+      it("should throw eServiceDescriptorNotFound if the descriptor doesn't exist", async () => {
         const eService: EService = {
           ...mockEService,
-          descriptors: [descriptor],
+          descriptors: [],
```

Comment:
I think it's useful to see at a glance that this `eService` doesn't have a descriptor, since this test has to throw an `eServiceDescriptorNotFound` error

Reply:
> I think it's useful to see at a glance that this `eService` doesn't have a descriptor, since this test has to throw an `eServiceDescriptorNotFound` error. Otherwise who's reviewing this test might not remember that the default `mockEservice` doesn't have any descriptors.

Usually, when something isn't explicitly added in a test the assumption is that it is not present (We do the same in QA, only declare what you add, we are not declaring what we are NOT adding).

In any case this is just my preference, if you think it's more clear in this way you feel free to leave it as it is.

Reply:
Let's see if @galales has any preferences here.